File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    let arr = ["aya", "ye", "woo", "ma"];
    let answer = [];
    
    for(let i=0; i<babbling.length; i++) {
        for(let j=0; j<arr.length; j++) {
            if(babbling[i]===arr[j] || babbling[i].includes(arr[j])) {
                babbling[i] = babbling[i].replace(arr[j], " ")
            }
        }
        answer.push(babbling[i]);
    }
    return answer.toString().replaceAll(" ", "").split(",").reduce((a, b) => (b==="" ? a+1 : a), 0);
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    babbling_set = {"aya", "ye", "woo", "ma"}
    cnt = 0

    for b in babbling:
        for c in babbling_set:
            if c in b:
                b = b.replace(c, " ")

        if b.isspace():
            cnt += 1

    return cnt